spec/models/game_spec.rb

```diff
@@ -46,7 +46,19 @@
         expect(Piece.where(type: piece, color:'white').count).to eq(number)
         expect(Piece.where(type: piece, color:'black').count).to eq(number)
       end
-      
+
+    end
+  end
+
+  describe '.check' do
+
+    before do
+      @king = FactoryGirl.create(:king, file: 4, rank: 4, color: :black_player_id)
+      @piece = FactoryGirl.create(:piece, file: 5, rank: 5, game: @king.game, color: :white_player_id)
```

Comment:
See the attachment in the above comment for how this should be setup.  For the most part, this works, though I removed the @piece variable since we want to test all the pieces and not just a generic one in each test case